libnavigation-base/src/main/java/com/mapbox/navigation/base/trip/model/roadobject/RoadObjectType.kt

```diff
@@ -75,6 +75,11 @@ object RoadObjectType {
      */
     const val JCT = 10
 
+    /**
+     * Not finished yet, see https://mapbox.atlassian.net/browse/NAVAND-1311
+     */
+    internal const val NOTIFICATION = 11
```

Comment:
@dzinad , @averkhaturau , as I know the notification feature isn't stable on server side and the API may change. Do I understand correctly that current implementation in NN works for current design and just stop working in case of changes on server side without crashes?